### ISSUE TITLE
Feature Request: Save to file

### ISSUE THREAD

**Issue by jefhai** [IvanMathy/Boop#278]
Can we have a save to file option? File -> Save… I love the scratchpad functionality but I also have had times I just need to save to a file and can’t find a save feature. I ended up opening another editor or terminal and saving to file.

**Comment by Altair-Bueno** [comment]
As a workaround you can:

1. Select all with <kbd>command + a</kbd>
2. Drag & drop to X folder

Most of the time i find it easier to use this trick on webpages/small texts because it's faster than export/create file (finder doesn't pop up), preserves metadata and behaves like macos's clipboard. I hope you find it useful

https://user-images.githubusercontent.com/67512202/130332493-a12830f5-164a-403a-a8b1-b640d07b72fe.mov

**Comment by grafst** [comment]
I think boop should do one thing well, as ist currently does.
It's not an editor...

**Comment by jefhai** [comment]
@grafst yeah save to file doesn’t make it an editor… I’m not asking for open file. Just a save out option. Tools should have less friction for their one thing well. And Boop right now is under developed by the maintainer and requires you to open another app and copy paste just to save out.

**Comment by grafst** [comment]
"save to file doesn’t make it an editor" fair, you are right.
